QUARKUS-5615.md

```diff
@@ -0,0 +1,30 @@
+## Links to the resources
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5615
+- Upstream documentation PR: https://github.com/quarkusio/quarkus/pull/46182
```

Comment:
PR link for deserializers functionality https://github.com/quarkusio/quarkus/pull/42954 is missing

Reply:
Added the link